python/cudf/bindings/sort.pyx

```diff
@@ -52,3 +52,28 @@ cpdef apply_sort(col_keys, col_vals, ascending=True):
         gdf_radixsort_plan_free(plan)
         free(c_col_keys)
         free(c_col_vals)
+
+cpdef apply_order_by(in_cols, out_indices, ascending=True, na_position=1):
```

Comment:
Is the prior apply_sort function still used? Should we keep it, if so, can we add a note of whether it’s a legacy or when each one is used?

Reply:
Removed the old `apply_sort` implementation from the Cython / Python, but there's still a segmented radix sort in there tied into the legacy groupby code that I don't want to fight with pulling out in this PR.